content/self-host/rustdesk-server-pro/smtp/Microsoft365/_index.fr.md

```
@@ -22,7 +22,7 @@ Pour la configuration SMTP generale, consultez [SMTP](../).
 | OAuth2 Client secret | La `Value` du secret cree sous `Certificates & secrets` |
 
 Cette capture montre ou saisir ces valeurs dans RustDesk :
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft 365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)